### PR TITLE
operator minio-operator-rhmp (5.0.14)

### PR DESCRIPTION
**Pull request by pjuarezd**
operator minio-operator-rhmp (5.0.14)
Signed-off-by: pjuarezd <pjuarezd@users.noreply.github.com>

### CODE REVIEW

**Comment by pjuarezd** [comment]
closes https://github.com/minio/operator/issues/2050

File: operators/minio-operator-rhmp/5.0.14/manifests/minio-operator-rhmp.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,785 @@
...
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/csi: "false"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    olm.skipRange: '<5.0.14'
```
Comment (acornett21) [inline operators/minio-operator-rhmp/5.0.14/manifests/minio-operator-rhmp.clusterserviceversion.yaml:97]: Based on the previous versions of how you release your operator, I do not think this is what you want. Normally you are use `replaces`, now that you are switching to be in a new catalog, you should remove this line and change your `replaces` that is normally at the end of the CSV to a `skips`

Comment (acornett21) [reply operators/minio-operator-rhmp/5.0.14/manifests/minio-operator-rhmp.clusterserviceversion.yaml:97]: More info on `skipRange` [here](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/#skiprange)

```
WARNING: As a consequence of using skipRange the skipped operator versions will be pruned from OLMs update graph and therefore will not be installable anymore by users with the spec.startingCSV property of Subscriptions. Use with caution. If you want to have direct (one hop) updates to this version from multiple previous versions but keep those previous versions available to users for install, always use skipRange in conjunction with replaces pointing to the immediate previous version of the operator version in question.
```

File: operators/minio-operator-rhmp/5.0.14/manifests/minio-operator-rhmp.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+    url: https://min.io
+  relatedImages:
+    - image: quay.io/minio/operator@sha256:db47d598488f10daf6a4d26431ab2b65e24f3dffbae0edb76bc44f63c6daf500
+      name: console
+    - image: quay.io/minio/operator@sha256:db47d598488f10daf6a4d26431ab2b65e24f3dffbae0edb76bc44f63c6daf500
+      name: minio-operator
+    - image: quay.io/minio/minio@sha256:ea2c70cda70860c7d10f04ce1df640d3abb995784cb7aeedef8d4baa53a5a113
+      name: minio-ea2c70cda70860c7d10f04ce1df640d3abb995784cb7aeedef8d4baa53a5a113-annotation
+  version: 5.0.14
+  skips: minio-operator-rhmp.v5.0.13
```
Comment (acornett21) [inline operators/minio-operator-rhmp/5.0.14/manifests/minio-operator-rhmp.clusterserviceversion.yaml:785]: `skips` is an array and must be formatted properly.
```
skips:
   - minio-operator-rhmp.v5.0.13
```

Comment (pjuarezd) [reply operators/minio-operator-rhmp/5.0.14/manifests/minio-operator-rhmp.clusterserviceversion.yaml:785]: got it, on it

**Pull request merged**